### PR TITLE
Fix l1 token upgrade

### PR DESCRIPTION
**Pull request by ElfoLiNk**

### CODE REVIEW

File: code/apim_global/policy.xml.tpl [hunk 1]
```diff
@@ -17,6 +17,7 @@
         <header>Content-Type</header>
         <header>Authorization</header>
         <header>Accept</header>
+        <header>X-CGN-TOKEN</header>
```
Comment (pasqualedevita) [inline code/apim_global/policy.xml.tpl]: lowercase?

Comment (AleDore) [reply code/apim_global/policy.xml.tpl]: Yep it's a better choice

Comment (ElfoLiNk) [reply code/apim_global/policy.xml.tpl]: lowercased

**Review by AleDore — APPROVED** [review]
LGTM

**Pull request merged**